### ISSUE TITLE
Example code does not work out of the box - awsx

### ISSUE THREAD

**Issue by TechplexEngineer** [pulumi/pulumi-hugo#2300]
File: [themes/default/content/docs/guides/crosswalk/aws/_index.md](https://www.pulumi.com/docs/guides/crosswalk/aws/)'

When running the sample code
```typescript
import * as awsx from "@pulumi/awsx";
import * as aws from "@pulumi/aws";

// Create a repository.
const repo = new awsx.ecr.Repository("my-repo");

// Build an image from the "./app" directory (relative to our project and containing Dockerfile),
// and publish it to our ECR repository provisioned above.
const image = new awsx.ecr.Image("image", {
    repositoryUrl: repo.url,
    path: "./app",
})

// Create an ECS Cluster
const cluster = new aws.ecs.Cluster("default-cluster");

// Create a load balancer on port 80 and spin up two instances of Nginx.
const lb = new awsx.lb.ApplicationLoadBalancer("nginx-lb");

const service = new awsx.ecs.FargateService("my-service", {
    cluster: cluster.arn,
    taskDefinitionArgs: {
        container: {
            image: image.imageUri,
            cpu: 512,
            memory: 128,
            essential: true,
            portMappings: [
                {
                    containerPort: 80,
                    targetGroup: lb.defaultTargetGroup,
                },
            ],
        },
    },
});

// Export the load balancer's address so that it's easy to access.
export const url = lb.loadBalancer.dnsName;
```
Error is generated:
```
Error: The default VPC does not have any private subnets. Set `assignPublicIp` to `true`, provide `networkConfiguration`, or add private subnets to the default VPC.
```

**Comment by roothorp** [comment]
Hey @TechplexEngineer, thanks for the issue! Apologies that this example isn't working - we recently released a 1.0 and are in the process of updating our docs. There's a functional example on the [ECR page](https://www.pulumi.com/docs/guides/crosswalk/aws/ecr/#consuming-a-private-repository-from-ecs) which is the same as this example, just adding assignPublicIp to avoid the error you're seeing - hopefully this helps. We're working to update the remaining pages as soon as we can!

**Comment by lukehoban** [comment]
This has been fixed.
